packages/suite/src/views/wallet/tokens/common/TokenList.tsx

```diff
@@ -291,14 +293,23 @@ export const TokenList = ({
                                                             : true,
                                                 },
                                                 {
-                                                    label: <Translation id="TR_UNHIDE_TOKEN" />,
+                                                    label: (
+                                                        <Translation
+                                                            id={
+                                                                tokenStatusType ===
+                                                                TokenManagementAction.SHOW
+                                                                    ? 'TR_UNHIDE_TOKEN'
+                                                                    : 'TR_HIDE_TOKEN'
+                                                            }
+                                                        />
+                                                    ),
                                                     icon: 'EYE_CLOSED',
```

Comment:
True but we have just closed or slashed.

Also I don't think this icon matches the button action since it says unhide but the icon is more like hide...
![Screenshot 2024-07-23 at 4 58 42 PM](https://github.com/user-attachments/assets/39557dfc-1780-45b0-a1e4-71233e8cfeae)

cc @Hermez-cz